src/test.js

```diff
@@ -378,7 +378,7 @@ Test.prototype = {
 					function( error ) {
 						message = "Promise rejected " +
 							( !phase ? "during" : phase.replace( /Each$/, "" ) ) +
-							" " + test.testName + ": " + ( error.message || error );
+							" \"" + test.testName + "\": " + ( error && error.message );
```

Comment:
In case of a rejection with a string, should this fall back to `error` like the old logic had done? Something like `( ( error && error.message ) || error )` maybe?